### DISCUSSION TITLE
Manual/class-SortingGroup

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#692] [Notes]
# Manual/class-SortingGroup

https://docs.unity3d.com/Manual/class-SortingGroup.html

**Comment by cmcpasserby** [comment]
When sorting groups are sorted based on your sort settings, but a nice side effect is its based on your transform position and not the pivot or centre like with a sprite. so this can be used to get a little more control over what point is used for sorting as well as grouping things.
